tests/integration/cli/tests/run.rs

```diff
@@ -81,6 +81,53 @@ fn test_cross_compile_python_windows() -> anyhow::Result<()> {
     Ok(())
 }
 
+#[test]
+fn run_whoami_works() -> anyhow::Result<()> {
+    let ciuser_token = std::env::var("WAPM_DEV_TOKEN").expect("no CIUSER / WAPM_DEV_TOKEN token");
+
+    let output = Command::new(get_wasmer_path())
+        .arg("login")
+        .arg("--registry")
+        .arg("wapm.dev")
+        .arg(ciuser_token)
+        .output()?;
+
+    if !output.status.success() {
+        bail!(
+            "wasmer login failed with: stdout: {}\n\nstderr: {}",
+            std::str::from_utf8(&output.stdout)
+                .expect("stdout is not utf8! need to handle arbitrary bytes"),
+            std::str::from_utf8(&output.stderr)
+                .expect("stderr is not utf8! need to handle arbitrary bytes")
+        );
+    }
+
+    let output = Command::new(get_wasmer_path())
+        .arg("whoami")
+        .arg("--registry")
+        .arg("wapm.dev")
+        .output()?;
+
+    let stdout = std::str::from_utf8(&output.stdout)
+        .expect("stdout is not utf8! need to handle arbitrary bytes");
+
+    if !output.status.success() {
+        bail!(
+            "linking failed with: stdout: {}\n\nstderr: {}",
+            stdout,
+            std::str::from_utf8(&output.stderr)
+                .expect("stderr is not utf8! need to handle arbitrary bytes")
+        );
+    }
+
+    assert_eq!(
+        stdout,
+        "logged into registry \"https://registry.wapm.dev/graphql\" as user \"ciuser\"\n"
+    );
+
+    Ok(())
+}
```

Comment:
I don't think this test is a good idea.

It assumes the `$WAPM_DEV_TOKEN` variable is set and that user's name is `ciuser`, so the only way for integration tests to pass locally is if you have the `ciuser` token on your computer, which is a massive no-no (no human developers should have access to CI tokens).

We're also trying to contact wapm.dev, which makes the test slow and flaky (e.g. wapm.dev could be having a bad day and always error out).

Reply:
Yeah well with network requests, that's kind of a given, otherwise it wouldn't be an integration test. In the worst case the test fails because wapm.dev is down, then we just need to restart the tests on GitHub.

The other part, I think I can require `WAPM_DEV_TOKEN ` if `GITHUB_TOKEN` is present (true for integration tests in the CI, false for running tests locally).